### PR TITLE
SCAL-157042 : Update reload function

### PR DESCRIPTION
**Pull request by Nirmay15**

### CODE REVIEW

**Review by girish-thoughtspot — APPROVED** [review]

File: src/utils/processTrigger.ts [hunk 1]
```diff
@@ -6,12 +6,13 @@ import { HostEvent } from '../types';
  *
  * @param iFrame
  */
-function reload(iFrame: HTMLIFrameElement) {
-    const oldFrame = iFrame.cloneNode();
-    const parent = iFrame.parentNode;
-    parent.removeChild(iFrame);
-    parent.appendChild(oldFrame);
-}
+export const reload = (iFrame: HTMLIFrameElement) => {
```
Comment (abdullasulaiman) [inline src/utils/processTrigger.ts:9]: export const reload = (iFrame: HTMLIFrameElement) => {
    const src = iFrame.src;
    iFrame.src = '';
    requestAnimationFrame(() => {
        iFrame.src = src;
    });
};
Can you please try with requestAnimationFrame and see if it works ?

**Review by abdullasulaiman — CHANGES_REQUESTED** [review]
Please see the comment

**Review by abdullasulaiman — APPROVED** [review]
lgtm

File: src/utils/processTrigger.ts (hunk 1 above)
Comment (Nirmay15) [reply src/utils/processTrigger.ts:9]: It is not reloading the whole frame, it is working as internal navigation

**Pull request merged**